packages/mobile/src/screens/chat-screen/ChatListItemSkeleton.tsx

```diff
@@ -0,0 +1,68 @@
+import { View } from 'react-native'
+
+import Skeleton from 'app/components/skeleton'
+import { makeStyles } from 'app/styles'
+
+import { useStyles as useChatListItemStyles } from './ChatListItem'
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
+  profilePicture: {
+    width: spacing(12),
+    height: spacing(12),
+    borderRadius: spacing(6)
+  },
+  userNameContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: spacing(3)
+  },
+  userName: {
+    height: spacing(4),
+    width: spacing(25),
+    borderRadius: spacing(12)
+  },
+  handle: {
+    height: spacing(3),
+    width: spacing(22),
```

Comment:
Just curious why these widths are hard-coded to specific things? Could the values exceed that width?

Reply:
Since this is a skeleton we're just showing a generic glowy section that's always gonna be the same size.

Reply:
Oh I missed that this was a skeleton, carry on!